doc/releases.md

```diff
@@ -81,6 +81,22 @@ Notes:
 - Version strings are listed below as _"vx.y.z"_. Substitute for the release
   version.
 
+### 0. Pre-release steps
+
+Before preparing a Node.js release, you must notify the Build Working Group at
+least twenty-four hours in advance of the expected release. Coordinating with
```

Comment:
Maybe `twenty-four hours` -> `one business day` so that weekend days don't count?